elixir_daisy/settings_local.template.py

```diff
@@ -1,3 +1,5 @@
+GLOBAL_API_KEY = 'insert something random here'
+
```

Comment:
maybe we should do the same as in lines 13 and 14?

Reply:
resolved in 35a8291ac7246f59b663f712b1b70acd339f6196